cowprotocol/accounting/cost_coverage/cost_coverage_2601653.sql

```diff
@@ -0,0 +1,117 @@
+with
+purchased_eth as (
+    select
+        block_time,
+        sum(units_bought) as fees
+    from cow_protocol_ethereum.trades
+    where
+        buy_token_address in (0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2, 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee)
+        and trader in (0x84e5c8518c248de590d5302fd7c32d2ae6b0123c, 0x9008D19f58AAbD9eD0D60971565AA8510560ab41)
+        and receiver = 0xa03be496e67ec29bc62f01a428683d7f9c204930
+        and block_date between (timestamp '{{start_time}}' + interval '12' hour) and (timestamp '{{end_time}}' + interval '12' hour)
+    group by block_time
+),
+
+-- WETH Transfer from settlement contract to rewards safe
+transferred_eth as (
+    select
+        evt_block_time as block_time,
+        sum(value / 1e18) as fees
+    from erc20_ethereum.evt_Transfer
+    where
+        "from" = 0x9008d19f58aabd9ed0d60971565aa8510560ab41
+        and to = 0xa03be496e67ec29bc62f01a428683d7f9c204930
+        and contract_address = 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2
+        and evt_tx_hash not in (select tx_hash from cow_protocol_ethereum.trades where block_date between (timestamp '{{start_time}}' + interval '2' day) and (timestamp '{{end_time}}' + interval '2' day))
+        and evt_block_time between (timestamp '{{start_time}}' + interval '2' day) and (timestamp '{{end_time}}' + interval '2' day)
```

Comment:
This also needs to change doesn't it (since we are also transferring WETH with every fee harvest)?

Reply:
Ohh good catch, thanks!